posthog/models/team.py

```diff
@@ -199,6 +199,13 @@ def _timezone_feature_flag_enabled(self) -> bool:
             "timezone-for-charts", distinct_id, groups={"organization": str(self.organization_id)}
         )
 
+    @cached_property
+    def behavioral_cohort_querying_enabled(self) -> bool:
+        distinct_id = self.organization.members.filter(is_active=True).first().distinct_id
+        return posthoganalytics.feature_enabled(
```

Comment:
Hmm, I think it's better to use `instance settings` here, than an actual feature flag? Since otherwise, this implies sending a request to the posthog app (`/decide`) every time we get to this path 😅 .

Otherwise, both options are similar. Even with instance settings, we can choose which teams to enable this for, (except % rollout), mod that around using the UI as well, and

Reply:
... and get rid of it quickly once we move everything over to new queries, likely within this week.

The main issue is: If it's a `cached_property`, we can't quickly turn it off and on. If it's not a `cached_property`, we're making `/decide` requests every time we want to calculate a cohort. Hence, the instance settings here seems like a pretty decent middle ground... And in this case, we'd rather shut it down quickly, and then when fine, roll out to everyone quickly.

Reply:
cool with me